### DISCUSSION TITLE
This can be a problem.

### DISCUSSION THREAD

**Discussion by Ndrinta** [nathanmarlor/foxess_modbus#130] [General]
I remember seeing that in [v1.4.0b7](https://github.com/nathanmarlor/foxess_modbus/releases/tag/v1.4.0b7) one of the points is:
- Added integration sensors to replace manually configured yaml (https://github.com/nathanmarlor/foxess_modbus/pull/98) @nathanmarlor

After seeing this today (HA 2023.4) I got spooked up that maybe if we don't clean up properly the config.yaml we may see some unexpected issues with the history.

![image](https://user-images.githubusercontent.com/4704172/230420984-6213e695-3521-4cb4-bbd5-3b0dc825989d.png)

There is a good way of doing this, I'm sure.
Nathan, can you publish the list of entities that will then be handled by the integration natively so we can be smart when updating it?

**Comment by canton7** [comment]
In a similar vein, I've been wanting to adopt and update [this page](https://github.com/StealthChesnut/HA-FoxESS-Modbus/wiki/Data-Register-Reference---H1-AC1) into our wiki, so we have an up-to-date reference for what we're actually doing.

**Reply by Ndrinta** [reply]
And [this ](https://github.com/StealthChesnut/HA-FoxESS-Modbus/wiki/RS485-Agent-Addresses) and [this ](https://github.com/StealthChesnut/HA-FoxESS-Modbus/discussions/116#discussioncomment-5479869) and [this](https://github.com/StealthChesnut/HA-FoxESS-Modbus/discussions/113) and ...
